src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -147,6 +147,26 @@
     short-summary: Show logs of an app instance, logs will be streamed when setting '-f/--follow'.
 """
 
+helps['spring-cloud app identity'] = """
+    type: group
+    short-summary: manage an app's managed service identity

```

Comment:
/s/manage/Manage

Reply:
Some summary has period in the end, but some are not.

Reply:
For some summary there is period in the end, but some are not.